builds.go

```diff
@@ -0,0 +1,359 @@
+//
+// Copyright 2016, Arkbriar
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package gitlab
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/url"
+	"time"
+)
+
+// ListBuildsOptions are options for two list apis
+type ListBuildsOptions struct {
+	Scope []string `url:"scope,omitempty", json:"scope,omitempty"`
+}
+
+// BuildsService handles communication with the ci builds related methods
+// of the GitLab API.
+//
+// GitLab API docs: http://docs.gitlab.com/ce/api/builds.html
+type BuildsService struct {
+	client *Client
+}
+
+// Runner represents a ci runner
+//
+// GitLab API docs: http://docs.gitlab.com/ce/api/builds.html
+type Runner struct {
```

Comment:
I don't think we should define `Runner` here, as `Runner` has it's own API: https://docs.gitlab.com/ce/api/runners.html

So until we have implemented that API and it's type, I prefer to use this type as an anonymous embedded struct (like you had before) instead.